.github/workflows/sanitizer-build-and-test.yaml

```diff
@@ -122,12 +118,13 @@ jobs:
           https://ftp.postgresql.org/pub/source/v${{ matrix.pg }}/postgresql-${{ matrix.pg }}.tar.bz2
         mkdir -p ~/$PG_SRC_DIR
         tar --extract --file postgresql.tar.bz2 --directory ~/$PG_SRC_DIR --strip-components 1
+        patch -F5 -p1 -d ~/$PG_SRC_DIR < test/postgres-asan-instrumentation.patch
```

Comment:
Could you add a comment here pointing to our internal wiki (as in the PR comment) or to the discussion on the PostgreSQL mailing list? This would allow a further reader better understand the context without knowing the PR or the commit message.

Reply:
Added.